docs/the-new-architecture/pillars-codegen.md

```diff
@@ -230,3 +230,60 @@ The **Codegen** for a Fabric Native Component contains a `MyFabricComponentManag
 Then, there is a layer of JNI C++ files that are used by Fabric to render the components. The basic element for a Fabric Component is the `ShadowNode`: it represents a node in the React abstract tree. The `ShadowNode` represents a React entity; therefore it could need some props, which are defined in the `Props` files and, sometimes, an `EventEmitter`, defined in the corresponding file.
 
 The **Codegen** also creates a `ComponentDescriptor.h`, which is required to get a proper handle on the Fabric Native Component.
+
+## The Codegen CLI
+
+**npx react-native codegen** [**--path** *path*] [**--platform** *string*] [**--outputPath** *path*]
+
+#### DESCRIPTION
+
+This command runs `react-native-codegen` for your project.
+The Codegen must be configured as described in [Configure Codegen](../new-architecture-library-intro#configure-codegen).
+
+The following options are available:
+
+- `--path` - Path to `package.json`. The default path is the current working directory.
+- `--platform` - Target platform. Supported values: `android`, `ios`, `all`. The default value is `all`.
+- `--outputPath` - Output path. The default value is the value difined in `codegenCofig.outputDir`.
```

Comment:
```suggestion
- `--outputPath` - Output path. The default value is the value defined in `codegenCofig.outputDir`.
```
Out of curiosity, what happens if the `outputDir` is not defined in the `package.json` and the `outputPath` parameter is not passed? Does it crash? What error message is displayed?
Not that we need to update the docs, but the error message should guide the user toward solving the issue themselves.

Reply:
It will just put the generated code in the default hardcoded locations:
iOS: `build/generated/ios`
Android: `android/app/build/generated/source`